pr-data.csv

```diff
@@ -435,7 +435,7 @@ https://github.com/apache/cloudstack,64262d943f012c40deac3cd87c67d44fdc727b52,ut
 https://github.com/apache/commons-collections,371313a2e4d1da5e0776419ce94bb8b644158a20,.,org.apache.commons.collections4.bag.CollectionBagTest.testCollectionToArray2,ID,,,https://issues.apache.org/jira/browse/COLLECTIONS-804
 https://github.com/apache/commons-collections,371313a2e4d1da5e0776419ce94bb8b644158a20,.,org.apache.commons.collections4.bag.HashBagTest.testCollectionToArray2,ID,,,https://issues.apache.org/jira/browse/COLLECTIONS-804
 https://github.com/apache/commons-collections,371313a2e4d1da5e0776419ce94bb8b644158a20,.,org.apache.commons.collections4.bag.PredicatedBagTest.testCollectionToArray2,ID,,,https://issues.apache.org/jira/browse/COLLECTIONS-804
-https://github.com/apache/commons-collections,371313a2e4d1da5e0776419ce94bb8b644158a20,.,org.apache.commons.collections4.bag.SynchronizedBagTest.testCollectionToArray2,ID,,,https://issues.apache.org/jira/browse/COLLECTIONS-804
+https://github.com/apache/commons-collections,371313a2e4d1da5e0776419ce94bb8b644158a20,.,org.apache.commons.collections4.bag.SynchronizedBagTest.testCollectionToArray2,ID,Accepted,https://github.com/apache/commons-collections/pull/336,https://issues.apache.org/jira/browse/COLLECTIONS-804
```

Comment:
Thanks for your contribution. Based on the discussion for your PR and the actual commit, your change is more `InspiredAFix` than `Accepted`. Please change the status.